app/main.py

```diff
@@ -2,0 +1,5 @@
+def copy_file(command: str) -> None:
+    comm, file, new_file = command.split()
```

Comment:
Consider using more descriptive variable names instead of `comm`, `file`, and `new_file`. This aligns with the checklist's recommendation for using descriptive and correct variable names.